### ISSUE TITLE
navigateTo failing on entry page

### ISSUE THREAD

**Issue by maxIrvine** [FlutterFlow/flutterflow-issues#1835]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

The navigateTo action is not working on the entry page on some IOS devices. It does not seem related to any specific device or IOS version. We've had users that have the same iPhone and same IOS but one experiences the issue and the other does not.

Other FF users also have the issue but it doesn't seem to be as frequent for them: https://community.flutterflow.io/authentication/post/entry-page-problem-Wh8Zq144tjSSY8p?highlight=cM6vxzcqkImKuyN

### Expected Behavior

When a user clicks on the 'Get Started' button on an entry page it navigates to the proper page.

### Steps to Reproduce

1. Run the project on an IOS device (it does not happen on the web)
2. It only occurs on 30-40% of devices, but interestingly, we are continuously denied by Apple's review as a result of the issue. It occurs a disproportionate amount of times on their devices.

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4slvHl289Nobxc+Kr9bcJR/TgvKUQ9RoI0scp/bxIgGIDzPOw+ZPXsXE9pTdSqd1ZIJ0SmmjkxwsLHvfP9UsBfCw2rRr59zcxuahfwQT6ge5OBOqiSdGtDP8ZVGhWX0aWJuRN4Iu9ocUYY7Ez7IN7LVgjtHu/FImdISq/LZO4=

### Context

1. We only get approved by Apple about 1 of every 5 submissions, which dramatically slows our development cadence.
2. Our users experience the issue and its hurting our AppStore rating

### Visual documentation

https://github.com/FlutterFlow/flutterflow-issues/assets/29580341/3a4fc245-7a26-455a-85c1-23695cdf8dfc

### Environment

```markdown
- FlutterFlow version: 4.0
- Platform: IOS
- Browser name and version: Brave 1.58.137
- Operating system and version affected: Ventura 13.4.1, IOS (no specific version)
```

**Comment by maxIrvine** [comment]
I believe this might be the same issue as this ticket: https://github.com/FlutterFlow/flutterflow-issues/issues/1697

**Comment by Habari1447** [comment]
Same bug for me

**Comment by hariprasadms** [comment]
Hello @maxIrvine - Thanks for submitting the issue. As you described above behaviour is specific to few devices and inconsistent, its a challenge to reproduce. I can mark the issue as a project specific and start investigating. However, we are not able to give an ETA. Meanwhile, if you got more details about reproducible steps, please update in the issue. Thanks

**Comment by victoriahuang1** [comment]
@maxIrvine can you confirm if the issue is fixed for you, now that the fix to #1697 has been deployed?

**Comment by victoriahuang1** [comment]
Closing, but feel free to reopen if you are still facing issues.
